_data/authors.yml

```diff
@@ -34,3 +34,10 @@ lucia_payo:
   bio: ""
   avatar: authors/lucia_payo.gif
   twitter: LuciaP_86
+
+henning_gross:
+  name: Henning Groß
+  web: http://www.upday.com
+  email: igaffai@gmail.com
+  bio: "Henning leads the technical team at upday as director engineering. He is still passionate about code and tries to work as operational as possible for someone in a management position (which can be painful for the team at times)."
```

Comment:
_Director Engineering_ actually is my title. There is no _of_... :) I capitalized the letters to emphasize...
I changed the sentence.
I adopted bio/bio_long.
It looks good now.